vault.yaml

```diff
@@ -30,8 +32,8 @@ pipeline:
         make "$plugin"
       done
 
-      # Skip with web ui build since it needs Nodejs ver 16 explicitly.
-      _gotags="vault"
```

Comment:
Genuinely curious: is it possible to package the ui-having variant as a subpackage? Does that even make sense? I could imagine someone only wanting the server components and not the UI, but I don't know if that's a real scenario since I've never used Vault.

I'll defer to you completely, whatever your answer is 😄

Reply:
I'm not sure how to split them out since they build it and bundle it into the binary :-/

Reply:
This matches upstream, so I'm fine with it.